config/application.yml.default

```diff
@@ -324,6 +324,8 @@ sp_handoff_bounce_max_seconds: 2
 show_user_attribute_deprecation_warnings: false
 otp_min_attempts_remaining_warning_count: 3
 system_demand_report_email: 'foo@bar.com'
+sp_issuer_user_counts_report_configs: '[]'
+team_ursula_email: ''
```

Comment:
**[nitpick]**	should we get rid of the `team_ursula_email` config? zach will add it back in with his email PR

Reply:
oh yeah sure I didn't realize that was what it was from